setup.py

```diff
@@ -27,6 +27,7 @@
     author_email="boris.muzellec@owkin.com",
     packages=find_packages(exclude=["tests*"]),
     install_requires=[
+        "anndata==0.8.0",
```

Comment:
I think it's better to require a fixed `anndata` version and check manually that everything keeps working when `anndata` is updated, to avoid `DeseqDatSet` being potentially broken by major `anndata` changes.

Reply:
OK, let's keep this question on the side and move forward to merge quickly